core/src/main/scala/org/http4s/AttributeMap.scala

```diff
@@ -85,6 +77,6 @@ object AttributeMap
 // type inference required less generality
 /** A map entry where `key` is constrained to only be associated with a fixed value of type `T`. */
 final case class AttributeEntry[T](key: AttributeKey[T], value: T) {
-  override def toString: String = key.name + ": " + value
+  override def toString: String = s"$key: $value"
```

Comment:
I don't know: how the toString will provide the name and its hashcode, which is arguably better than it was before. I agree though: it would be nice if we could get the name from the value it gets assigned to.
```scala
scala> AttributeKey[Int]
res0: org.http4s.AttributeKey[Int] = org.http4s.AttributeKey@3c7d90c5
```